src/assignment.rs

```diff
@@ -2,3 +2,15 @@ use crate::common::*;
 
 /// An assignment, e.g `foo := bar`
 pub(crate) type Assignment<'src> = Binding<'src, Expression<'src>>;
+
+impl<'src> Display for Assignment<'src> {
+  fn fmt(&self, f: &mut Formatter) -> Result<(), fmt::Error> {
+    write!(
+      f,
+      "{}{} := {}",
+      if self.export { "export " } else { "" },
+      self.name,
+      self.value
+    )
```

Comment:
How about this:

```suggestion
    if self.export {
      write!(f, "export ")?;
    }
    write!(
      f,
      "{} := {}",
      self.name,
      self.value
    )
```

It's two calls to `write!(…)`, but a little bit clearer.